packages/website/docs/docs/image-toolbar.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
My main feedback on the docs is that it's now all centered around the Toolbar. This works for now, but is not very future proof if we soon plan to add:

- drag and drop: in that case, image / upload functionality is not related to the toolbar anymore, since you can add an image without opening the toolbar
- support for other files (arbitrary files, or videos)

Worth considering to already take this into account, or to migrate by then

Reply:
Agreed, I think when we implement drag and drop or other file types we'll want to move the docs for uploading to a different section, and have the Image Toolbar section reference it. I'm not sure we want to split them yet though, since right now the image toolbar is the only thing that does file uploading.